### PR TITLE
Create pull request template

### PR DESCRIPTION
**Pull request by aasthakasera**
Add pull_request_template.md

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Pull Request Template
```
Comment (DreamyPhobic) [inline .github/pull_request_template.md]: Remove this line

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
```
Comment (DreamyPhobic) [inline .github/pull_request_template.md:5]: move this to top.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## How Has This Been Tested?
```
Comment (DreamyPhobic) [inline .github/pull_request_template.md]: remove this section and add a section for Screenshots and other details.

**Review by DreamyPhobic — CHANGES_REQUESTED** [review]

**Comment by DreamyPhobic** [comment]
You have also committed issue template in this PR... You should made another PR for that.

File: .github/pull_request_template.md (hunk 2 above)
Comment (aasthakasera) [reply .github/pull_request_template.md:5]: Ok, i will do as soon as possible.

**Comment by aasthakasera** [comment]
Sir, actually I had made it separately but it was done automatically.

**Comment by aasthakasera** [comment]
How can I fix that, should I make a second PR after some time.

**Comment by DreamyPhobic** [comment]
contact me at zulip.

**Pull request closed without merging**